lib/mutant/integration/rspec.rb

```diff
@@ -35,6 +37,7 @@ class Rspec < self
       #
       def initialize(*)
         super
+        @all_expression = expression_parser.call(ALL_EXPRESSION)
```

Comment:
I think I want to always used the shortest form. I'll fix the `.call` ones to be `.`. Good catch.